Code/Editor/IEditorImpl.cpp

```diff
@@ -124,7 +126,10 @@ CEditorImpl::CEditorImpl()
     m_pUndoManager = new CUndoManager;
     m_pToolBoxManager = new CToolBoxManager;
     m_pSequenceManager = new CTrackViewSequenceManager;
-    m_pAnimationContext = new CAnimationContext;
+
+    IMovieSystem* movieSystem = nullptr;
+    Maestro::MovieSystemRequestBus::BroadcastResult(movieSystem, &Maestro::MovieSystemRequestBus::Events::GetMovieSystem);
+    m_pAnimationContext = new CAnimationContext(movieSystem);
```

Comment:
If this is the only place where CAnimationContext is instantiated, can we move the initialization of `m_movieSystem` in there instead of passing it on?